src/gui/pedalboard/cables/Cable.cpp

```diff
@@ -101,8 +101,8 @@ void Cable::repaintIfNeeded (bool force)
         cablePath = std::move (createdPath);
 
         auto cableBounds = cablePath.getBounds().toNearestInt();
-        cableBounds.setY (cableBounds.getY() - roundToInt (std::ceil (2.0f * minCableThickness)));
-        cableBounds.setHeight (cableBounds.getHeight() + roundToInt (std::ceil (4.0f * minCableThickness)));
+        cableBounds.setY (cableBounds.getY() - roundToInt (std::ceil (4.0f * minCableThickness)));
```

Comment:
@jatinchowdhury18 Is there a better way or metric to use for this bound? This is the smallest bound I found where everything was redrawing properly

Reply:
Yeah, that's probably as good as we can do... I'll keep thinking about it though.